node_server/src/main/routes/users.ts

```diff
@@ -42,12 +42,22 @@ function createUsersRouter(db: typeof mongoose): Router {
   const Project: ProjectModel = createProjectModel(db);
   const Task: TaskModel = createTaskModel(db);
 
+  router.options('/', (req, res, next) => {
+    res.setHeader('Access-Control-Allow-Credentials', 'true');
+    next();
+  });
+
   router.get('/', (req, res) => {
```

Comment:
It would be nice to have some swagger documentation on this route now that it behaves differently. But it is okay if not for now.

Reply:
Trying to write this now, unsure of how to describe the cookie -> session relationship in swagger :/

Reply:
It looks like it could be specified by a cookie Parameter Object according to the open API 3.0 spec: https://swagger.io/specification/ search "Parameter Object" on that page.

Reply:
Take a look at https://github.com/PointSpire/PointSpire/commit/40d19586c085e083b9d34866b85525147cc71ee0
referenced this https://swagger.io/docs/specification/authentication/cookie-authentication/